e2e/windows/e2e-scenario.sh

```diff
@@ -159,13 +156,12 @@ jq --argjson JsonForVnet "$WINDOWS_VNET" \
     --argjson JsonForSKU "$WINDOWS_SKU" \
     --argjson JsonForNetwork "$NETWORK_PROPERTIES" \
     --argjson JsonForOSDisk "$WINDOWS_OSDISK" \
-    --arg ValueForImageExternalID "$IMAGE_EXTERNALID" \
     --arg ValueForImageReference "$IMAGE_REFERENCE" \
     --arg ValueForAdminPassword "$WINDOWS_PASSWORD" \
     --arg ValueForCustomData "$CUSTOM_DATA" \
     --arg ValueForCSECmd "$CSE_CMD" \
     --arg ValueForVMSS "$DEPLOYMENT_VMSS_NAME" \
-    '.parameters += $JsonForVnet | .parameters += $JsonForLB | .parameters.galleries_AKSWindows_externalid.defaultValue=$ValueForImageExternalID | .resources[0] += $JsonForIdentity | .resources[0] += $JsonForSKU | .resources[0].properties.virtualMachineProfile.storageProfile+=$JsonForOSDisk | .resources[0].properties.virtualMachineProfile.networkProfile.networkInterfaceConfigurations[0] += $JsonForNetwork | .resources[0].properties.virtualMachineProfile.storageProfile.imageReference.id=$ValueForImageReference | .resources[0].properties.virtualMachineProfile.osProfile.adminPassword=$ValueForAdminPassword | .resources[0].properties.virtualMachineProfile.osProfile.customData=$ValueForCustomData | .resources[0].properties.virtualMachineProfile.extensionProfile.extensions[0].properties.settings.commandToExecute=$ValueForCSECmd | .parameters.virtualMachineScaleSets_akswin30_name.defaultValue=$ValueForVMSS' \
```

Comment:
could we also delete galleries_AKSWindows_externalid in [e2e/windows/windows_vmss_template.json](https://github.com/Azure/AgentBaker/blob/cb86e406100687b82f50ac03d2d248ef21861c64/e2e/windows/windows_vmss_template.json#L9)?

Reply:
Missed this part before. Thanks for the reminder.